encoding/scenefiltering.md

```diff
@@ -60,16 +60,17 @@ time. At this point, your script should look something like this:
 import vapoursynth as vs
 core = vs.get_core()
 
-src = core.lsmas.LWLibavSource("source.mkv")
-filtered = default_filtering(src)
+src = core.lsmas.LWLibavSource("source.m2ts")
+filtered = core.resize.Bilinear(src, width=1280, height=720)
```

Comment:
I actually find it weird to call the resized variant `filtered` (even though you say it's the "base filtering") when you do debanding, denoising and aa later. Maybe just call it `resized`?